index.js

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env node
+
+const core = require("@actions/core");
+const github = require("@actions/github");
```

Comment:
As far as I can see we're only using the exported `context` member here, so why not import only that?
```suggestion
const { context } = require("@actions/github");
```